src/extensions/default/JavaScriptCodeHints/unittests.js

```diff
@@ -875,11 +875,11 @@ define(function (require, exports, module) {
             it("should add new method on String .prototype", function () {
                 var start = { line: 37, ch: 0 };
                 var testPos = { line: 40, ch: 12 };
-                testDoc.replaceRange("String.prototype.times = function (count) {\n" + "\treturn count < 1 ? '' : new Array[count + 1].join(this);\n};\n\"hello\".time", start, start);
+                testDoc.replaceRange("String.prototype.times = function (count) {\n" + "\treturn count < 1 ? '' : new Array[count + 1].join(this);\n};\n\"hello\".tim", start, start);
```

Comment:
@nethip Yes, that's an intentional change.

That's the string typed in editor to filter js code hints. In previous test, ".time" would trigger a list with "times" and "trimLeft". But, "trimLeft" was dropped from the new version of **acorn**, so I changed test to instead use ".tim" to trigger list of "times" and "trim".